Add release process to README (#69)
ome/omero-gallery: README.rst

        {"key":"Name", "regex": "^(.*?)-.*?(.)$", "template": "$1$2"},
    ]'

Release process
---------------

> - review and update the CHANGELOG

Reviewer: ```suggestion
- review and update the [CHANGELOG](https://github.com/ome/omero-gallery/blob/master/CHANGELOG.md)
```

> - run `bumpversion release` to remove the dev suffix and create a signed tag

Reviewer: ```suggestion
- run ``bumpversion release`` to remove the dev suffix and create a signed tag
```

> - run `bumpversion --no-tag patch` to bump the version to the next dev suffix

Reviewer: ```suggestion
- run ``bumpversion --no-tag patch`` to bump the version to the next dev suffix
```

This one and the previous above did not render nicely, so trying to add 2 backticks.

> - push the newly created tag and `master` to `origin`. e.g. `git push origin master v3.3.3`

Reviewer: ```suggestion
- push the newly created tag and ``master`` to ``origin``. e.g. ``git push origin master v3.3.3``
```

> - the Travis CI build for the tag will also include a PyPI deployment step

Reviewer: ```suggestion
- the Travis CI build for the tag includes a PyPI deployment step, so no need to deploy on PyPi manually
```

        {"key":"Name", "regex": "^(.*?)-.*?(.)$", "template": "$1$2"},
    ]'

Release process
---------------

- review and update the CHANGELOG
- run `bumpversion release` to remove the dev suffix and create a signed tag
- run `bumpversion --no-tag patch` to bump the version to the next dev suffix
- push the newly created tag and `master` to `origin`. e.g. `git push origin master v3.3.3`
- the Travis CI build for the tag will also include a PyPI deployment step

> https://github.com/ome/omero-gallery/blob/352a9bcc227bf35ca63a3735b19d1a641dab0af5/.travis.yml#L17-L23

Reviewer 2: I probably wouldn't include this link in the README.
